### DISCUSSION TITLE
Group Assignment #1

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#4] [Assignments]
## Due 7/2 at 1pm PST
Post the link to your group repository alongside group members here to complete [group assignment #1](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_01/Materials/group_assignment.md)

Ex. Albert, Yoh: https://github.com/albertkun/my-example-project/

**Comment by emilysuh** [comment]
Emily Suh and Ronnie Blanche: https://github.com/emilysuh/alphabetmafia

**Reply by albertkun** [reply]
Nice use of markdown!!!!

**Comment by hrfrahn** [comment]
Harry Frahn & Sara Perez: 
https://github.com/hrfrahn/191A-GroupProject

**Comment by luigimagdamit** [comment]
LuigiAnton Magdamit & Abita Venkatesh: 
https://github.com/LuigiMagdamitUCLA/musical-lamp/tree/main

**Comment by edmondywen** [comment]
Cynthia Orozco & Edmond Wen:
https://github.com/edmondywen/asiaam-191a-la-heat-disparities

**Comment by eandrusenki** [comment]
Liz Andrusenki & Irvin Ramirez
https://github.com/eandrusenki/wehearseebelieveyou
